composer.json

```diff
@@ -16,7 +16,7 @@
         "rss": "https://github.com/laminas/laminas-recaptcha/releases.atom"
     },
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
+        "php": "~8.1.99 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

Comment:
update config platform instead under config > platform

Reply:
see section https://github.com/laminas/laminas-recaptcha/blob/6f5c05905f82b09a81faf90233f3eaf67ead891f/composer.json#L52-L54

Reply:
Thanks @samsonasik for pointing out the mistake.

Reply:
The issue is resolved. cheers